src/content/drafts/eslint-community-org.md

```diff
@@ -0,0 +1,131 @@
+of maintenance.  
+The Community Core team (which currently consists of
+[`@aladdin-add`](https://github.com/aladdin-add),
+[`@ota-meshi`](https://github.com/ota-meshi),
+[`@voxpelli`](https://github.com/voxpelli) and myself
+[`@MichaelDeBoey`](https://github.com/MichaelDeBoey)) and all package maintainers
+will make sure these packages stay up-to-date with newer ESLint releases, and
+they don't hold the wider community back by depending on a single person's
+GitHub or npm account.
+
+You might have noticed that some of your most beloved ESLint-related packages
+have already been moved over to or been forked by the `eslint-community`
+organization. A full list of the current projects include:
+
+* [`eslint-formatter-codeframe`](https://github.com/eslint-community/eslint-formatter-codeframe)
+  and
+  [`eslint-formatter-table`](https://github.com/eslint-community/eslint-formatter-table)  
+  We took these over from [@fregante](https://github.com/fregante)
+* [`eslint-plugin-eslint-plugin`](https://github.com/eslint-community/eslint-plugin-eslint-plugin)  
+  [@aladdin-add](https://github.com/aladdin-add),
+  [@bmish](https://github.com/bmish) and
+  [@not-an-aardvark](https://github.com/not-an-aardvark) transferred this repo
+  to the new org, but they're still maintaining it
+* [`eslint-plugin-promise`](https://github.com/eslint-community/eslint-plugin-promise)  
+  We took this one over from [@xjamundx](https://github.com/xjamundx)
+* [`eslint-plugin-security`](https://github.com/eslint-community/eslint-plugin-security)  
+  We took this one over from the
+  [@nodesecurity](https://github.com/nodesecurity) team, but
+  [@nzakas](https://github.com/nzakas) is still maintaining it
+* [`eslint-utils`](https://github.com/eslint-community/eslint-utils) and
+  [`regexpp`](https://github.com/eslint-community/regexpp)  
+  We created a fork from [@mysticatea](https://github.com/mysticatea)'s repos and
+  published them under the
+  [`@eslint-community` npm organization](https://npmjs.com/org/eslint-community).
+
+We also intend to fork, bring up to date, and publish the following packages
+originally authored by [@mysticatea](https://github.com/mysticatea) as well:
+
+* [`eslint-plugin-es`](https://github.com/mysticatea/eslint-plugin-es)  
+  We'll work on getting all
+  [`eslint-plugin-es-x`](https://github.com/eslint-community/eslint-plugin-es-x)
+  features merged as well
+* [`eslint-plugin-eslint-comments`](https://github.com/mysticatea/eslint-plugin-eslint-comments)
+* [`eslint-plugin-node`](https://github.com/mysticatea/eslint-plugin-node)  
+  We'll work on getting all
+  [`eslint-plugin-n`](https://github.com/eslint-community/eslint-plugin-n)
+  features merged as well
+
+Of course, we don't want to stop there.
+
+If you think your project would be a good fit for the new community org, *or*
+you're depending on a project that's currently unmaintained and want to have the
+community to take it over, you can contact the Community Core team on
```

Comment:
Please double-check capitalization throughout. In some places we are using "Community Core team" and in others "community core team". I'm fine with either, let's just be consistent.